### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by pawamoy** [mkdocs/mkdocs#3673] [Announcements]
Just a quick announcement to welcome you here!

You can start by introducing yourself in the [Introduction posts discussion](https://github.com/mkdocs/mkdocs-team-space/discussions/11), though that's optional :slightly_smiling_face: 

Feel free to share your ideas and opinions on existing discussions, or create new ones :slightly_smiling_face: 

Some recommendations:

- Be sure to stay on topic: comment or share new ideas on issue and PRs management, team organization, maintainership, or new project ideas that could impact other projects and their maintainers. Technical issues and general Q/A should continue being discussed at https://github.com/mkdocs/mkdocs/discussions.
- Be sure to use the available categories when creating new discussions (General, Ideas, Maintenance, Meta).
- Be sure to use the threaded feature of discussions when replying to a specific comment.

Let us know if we can do anything to improve the onboarding process!

**Comment by squidfunk** [comment]
I want to add that we should not let this space turn into a wishlist of things. The reason why we started this space is to put MkDocs maintenance on more shoulders and better understand the needs of plugin and extension authors. Thus, topics should be centered around how we can improve MkDocs to better cater to its ecosystem. Thus, I propose the following rule of thumb: when you're proposing something, you're stating that you are committing to invest time into it ☺️

**Reply by Andre601** [reply]
What if someone wants to brainstorm an idea that is not yet ready for public discussions (not suited for the mkdocs/mkdocs discussions) or can't yet be worked on by them or others (Or they simply can't work on it themself due to lack of knowledge)?

I'm sure a few people here will have great ideas that could be made for MkDocs, but lack the experience and skill to work on it themself, so what other kind of "investment" could they do then? Just share the knowledge? Testing?

**Reply by squidfunk** [reply]
@pawamoy and I intended this as a private space for contributors, as well as plugin and extension authors, to talk about concrete changes to the code base and or improvements to interop within the ecosystem. It is not meant as a general "would be cool if" space – this is what [the public discussion board](https://github.com/mkdocs/mkdocs/discussions) is for, so I would kindly ask you to use the public board. We can all read and answer on the public board. This private space is a subset which is meant to allow us to focus on the hard things that lie in front of us, not to dream about what we could do after solving all of those imminent challenges.

**Comment by Andre601** [comment]
I would like to propose 2 small things in relation to these discussions:

1. Pin this welcome post, so that new members may see it straight away.
2. Maybe add labels, so that discussions can be categorized further, i.e. with current "states" (In Progress, Planned, Paused, etc) and importance levels (Low, Medium, High).

**Reply by squidfunk** [reply]
I just pinned this post. And yes, we also need to settle on a label system, which is a whole other discussion, but maybe we can start with categories as is already discussed in mkdocs/mkdocs#3666.

On another important note: we're just starting the discussion, inviting people who want to take part, to circle around a process that specifies how we want to work and how to decide upon things together. I expect the first few weeks to be an expansion phase, which will be followed by a consolidation phase where we will establish processes etc. Once this is done, we will write up a guide that explains how we work. That's my proposal of how we should proceed to actually make progress besides plain talking.
